namespaces.go

```diff
@@ -33,15 +33,23 @@ type Cache struct {
 	Resource     string `json:"resource"`
 }
 
+// Credential generation information
+type CredentialGeneration struct {
+	Issuer        *string  `json:"issuer"`
```

Comment:
Why are these pointers?

Reply:
These must be pointers because it's an optional field in the schema.  The non-pointer version will throw an error in the Vault case, for example.